examples/composer-shared-vpc/terraform.tfvars

```diff
@@ -0,0 +1,37 @@
+org_id                    = ""
```

Comment:
I'd rename this file with a .sample at the end

Reply:
Done